torch/_dynamo/output_graph.py

```diff
@@ -1199,7 +1199,7 @@ def wrap_name(module_key: str) -> VariableTracker:
 
             def wrap_name(module_key: str) -> VariableTracker:
                 return SymNodeVariable.create(
-                    self,
+                    self.root_tx,
```

Comment:
Side note: this is the best example I have seen yet as to why I believe these PRs (and type checking) are important

Prior to this, we were passing self (which is an OutputGraph) into the create function.  How this did not cause runtime errors I do not know, because in `SymNodeVariable.create` we access `tx.output_graph` which would be None here and potentially cause a runtime exception deep in the stack